.changeset/green-news-boil.md

```diff
@@ -0,0 +1,5 @@
+---
+"@chakra-ui/avatar": patch
+---
+
+Fixed srcSet prop not being used in AvatarImg
```

Comment:
What do you think about providing our users more info about the srcSet attribute?

```suggestion
Added the prop `srcSet` to the `<Avatar />` and `<AvatarImage />` components to allow responsive image sources.
[Read more on MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-srcset)
```

Reply:
I didn't even think about it, thanks Tim!